arches_component_lab/src/arches_component_lab/widgets/api.ts

```diff
@@ -0,0 +1,8 @@
+import arches from "arches";
+
+export async function fetchLanguages() {
```

Comment:
If there's a concern about duplicate calls here, let's just set the appropriate cache headers on the backend.

Reply:
oh wow this is one of the few times I agree with caching 😂